herbert_core/classes/MPIFramework/messages/MESS_NAMES.m

```diff
@@ -29,36 +111,38 @@
             % usage:
             % id = MESS_NAMES.mess_id('completed')
             % or
-            % ids = MESS_NAMES.mess_id('completed','running','started')
+            % ids = MESS_NAMES.mess_id('completed','log','started')
             %
             % where id-s is the array of message id-s (tags)
             %
+            name2code_map=MESS_NAMES.name_tag_maps();
```

Comment:
'2'?!

Reply:
name to(2) code map ? is it not a common acronym?

Reply:
it's not one I like -  `name_to_code` and `code_to_name` are clearer. We're not in FORTRAN and limited to 6 character names

Reply:
on the verge of acceptance.... despite we are not limited by keyword length, an abbreviation name2code_map reads (IMHO) much easier then the name_to_code_map....